src/ApiRequestPanel.js

```diff
@@ -447,6 +474,34 @@ export class ApiRequestPanel extends AmfHelperMixin(EventsTargetMixin(HeadersPar
     this.requestUpdate('selectedServerValue', old);
   }
 
+  get showServerSelector() {
+    return this._showServerSelector;
+  }
+
+  set showServerSelector(value) {
```

Comment:
Why are we overriding getter and setter? Can we just remove them?

Reply:
Yes we can

Reply:
What I meant yesterday is to replace a function call in a template with a getter. It would do exactly the same and return the same value but instead of function you use a getter. General rule is not to use function calls in templates.